vss-extension.json~

```diff
@@ -10,11 +10,11 @@
       "Public"
   ],
 =======
-  "version": "0.1.25",
-  "name": "Kubernetes extension",
-  "description": "Kubernetes extension with k8s endpoint. Which enable kubectl command",
-  "publisher": "TsuyoshiUshio",
->>>>>>> parent of 00763c9... add support to download kubectl
+  "version": "0.1.45",
+  "name": "Kubernetes Ostrich extension",
```

Comment:
Thanks, I've never noticed. What is 'Ostrich'? Kubernetes release name or something? At least, we need to avoid version problem. I don't want to update everytime. haha.